README.md

```diff
@@ -1,2 +1,20 @@
 # HW01
-Edit the readme
+## Ariana F. Tortolani 
+### 07/05/2020
+
+
+## About Me
+Hi!  My name is Ariana and I am from Hope, Rhode Island.  I am an incoming Ph.D. Student at **The University of Chicago** in the *Committee on Computational Neuroscience*.  Currently, I am interested in studying motor control and proprioception.  I am interested in how these properties work together through sensori-motor integration to allow an individual to successfully perform goal-directed behaviors such as reaching out to pick up an object.
```

Comment:
Hi Ariana, I'm Ben, another incoming first year in the neurobiology program. Looking forward to working with you and the other teammates during this course!